libflux/flux-core/src/doc/mod.rs

```diff
@@ -11,15 +11,15 @@ use std::{
 
 use anyhow::{bail, Result};
 use derive_more::Display;
-use lazy_static::lazy_static;
+use once_cell::sync::Lazy;
```

Comment:
`once_cell` is newer and takes advantage of `const fn` instead of macros which I think is prefereable. The `once_cell` API is on track of being merged into rusts standard library as well so it is easier to swap out once that is stabilized https://github.com/rust-lang/rust/issues/74465